### DISCUSSION TITLE
Rebus Warnings: Closed channel detected - consumer will be disposed

### DISCUSSION THREAD

**Discussion by wfurney** [elsa-workflows/elsa-core#3075] [Q&A]
I am using RabbitMq for Rebus and lately I have been getting a lot of "Closed channel detected - consumer will be disposed" warnings in my logs. I didn't notice any warnings of this type during my first few months of use. I have not used Rebus or RabbitMq much before, does this imply that something is unhealthy? Is there some sort of maintenance that I should be performing on RabbitMq?

**Comment by sfmskywalker** [comment]
I've not seen this before or heard about it. A quick google search doesn't produce too many great results either.
Are you still seeing this frequently?

**Reply by wfurney** [reply]
Yes, I see around 10 of these warnings a day. Most days they show up but not every day. Otherwise Elsa appears to be working correctly as far as I can tell.
